generator/composer.json

```diff
@@ -1,6 +1,6 @@
 {
   "require-dev": {
-	  "php" : "^7",
+    "php" : "^7",
```

Comment:
The indent seems different from other lines

Reply:
Done.